std-bits/table/src/main/java/org/enso/table/data/table/Column.java

```diff
@@ -116,12 +118,16 @@ public Column rename(String name) {
    * @param items the items contained in the column
    * @return a column with given name and items
    */
-  public static Column fromItems(String name, List<Object> items) {
-    InferredBuilder builder = new InferredBuilder(items.size());
-    for (Object item : items) {
-      builder.appendNoGrow(item);
+  public static Column fromItems(String name, Value items) {
```

Comment:
Even with `EnsoDate` introduction this change still remains necessary, @jdunkerley & @kustosz. Either the Java requests `List<LocalDate>` or it gets `Value` and needs to do its conversion manually later. Nothing else can work according to my knowledge of Truffle.

Reply:
Take a `List<Value> items` as param: will simplify iteration

Reply:
Done in [922ead2](https://github.com/enso-org/enso/pull/3559/commits/922ead222c615ed57aca49ab1fff0c6b8accb0b9)